guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -0,0 +1,174 @@
+
+. Click the *Association* tab.
+
+. Select the operating systems that you want to register.
+
+. Navigate to *Hosts* > *Operating Systems* and click the operating system that you want to register.
+
+. Click *Submit*.
+
+. Click the *Templates* tab.
+
+. From the *Registration template* list, select *Linux registration default*.
+
+. Click *Submit*.
+
+. Navigate to *Hosts* > *All Hosts* > *Register Host*.
+
+. Optional: If you want to register hosts to {SmartProxy}, enter the following command on the {SmartProxy} to enable required modules:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# foreman-installer --foreman-proxy-registration \
+--foreman-proxy-templates \
+--foreman-proxy-template-url 'http://{smartproxy-example-com}'
+----
+
+. What fields are required for Satellite and Foreman?
+
+. Optional: From the *Host Group* list, select the host group to associate the hosts with.
+
+. From the *Operating System* list, select the operating system of hosts that you want to register.
+
+. Optional: If you want to register hosts to a {SmartProxy}, from the *{SmartProxy}* list, select the {SmartProxy} to register hosts to.
+
+ifeval::["{build}" == "satellite"]
+. Optional: If you do not want to register hosts with Insights, from the *Insights* list, select *No*. What are the defaults for those two?
+endif::[]
+
+. Optional: If you do not want to deploy {Project} SSH keys to hosts, from the *Remote Execution* list, select *No*.
+
+. Optional: {Project} uses the JSON Web Tokens (JWT) for authentication.
+This duration of this token defines how long the generated `curl` command works.
+If you want to change the duration of the token, enter the required duration to the *Token lifetime (hours)* field.
+You can set the duration to any number of hours.
++
+Note that {Project} uses the permissions of the user who generated the `curl` command for authorization of hosts.
+If user loses or gains additional permissions, the permissions of the JWT change too.
+Therefore, do not delete, block, or change permissions of the user during the token duration.
+
+. Optional: What is remote execution interface?
+
+ifeval::["{build}" != "satellite"]
+. In the *Activation Key(s)* field, enter one or more activation keys to assign to registered hosts.
+endif::[]
+
+. Click *Generate command*.
+
+. Copy the generated `curl` command to enter it on the hosts.
+
+. On the hosts that you want to register, enter the generated `curl` command as `root`.
+
+.SSL
+For calling Foreman endpoints via https CA authority must be installed on the host.
+Path to CA file can be found in Settings > Authentication > SSL CA file.
+
+If you don't want to upload & install CA on the host, you can run first `curl` command
+with `---insecure` parameter, like this:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+curl -X GET --insecure https://foreman.example.com/register | bash
+----
+
+This make first call insecure, however Global Registration template will
```

Comment:
Sound like an attacker might fetch the CA from this call and mess something up. Is it that scary?

Reply:
Well, to be honest, I don't know. But if attacker can fetch CA from the call, that means two things:

* He / She has access to the registered machine and from logs can get the command (with auth token)
* Or has access credentials to the API and can call /register endpoint

That being sad, having access to CA is the last thing we should worry about, IMHO.

Reply:
IDK here, an atacker accessing a registered host is not something I would like to see :) But I see your point, I will document both methods and details about the unsafety.